www/shared/test-utils/modules/react-app-tree/AppTree.test.js

```diff
@@ -1,38 +1,38 @@
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import AppTree from './AppTree';
 
 it('should render children correctly', () => {
-    const { container } = render(
+    render(
         <AppTree>
             <div>foo</div>
         </AppTree>,
     );
 
-    expect(container).toHaveTextContent('foo');
+    expect(screen.getByText('foo')).toBeInTheDocument();
 });
 
 it('should correctly setup IntlProvider', () => {
-    const { container } = render(
+    render(
```

Comment:
I think we are rendering AppTree inside another, because the custom render uses AppTree as well. Maybe, for all tests in this file, we should pass the options.wrapper equal to a simple div?

Reply:
Oh yes, we are! I wil do this adjustment. Thanks for pointing that out 🙌

Reply:
Actually, this is using the render from the library itself, not the custom one. Should we keep it as it is or pass the wrapper option as you suggested?

Reply:
It’s good as is!